### PR TITLE
Add demo page, change `uni-page` to `uni-pages`, fix custom properties in polyfilled browsers

### PR DESCRIPTION
**Pull request by tomalec**
https://github.com/Starcounter/uniform.css/issues/3

### CODE REVIEW

**Comment by tomalec** [comment]
@warpech  I was not sure whether I should even show light DOM demos, or should I just put everything, all the classes into shadow roots, and present it that way.

**Comment by tomalec** [comment]
I fixed some issues "by-the-way" but the layout still looks different than in Anders' designs. I'd like to add this demo and track those individually.

**Comment by tomalec** [comment]
In the planning meeting, we decided to remove `uniform.css` from the light dom of the main document, and show the demos of a simple one, and the complex that styles distributed elements, as well as elements from the same tree.

**Comment by alshakero** [comment]
Should I review this now?

**Comment by warpech** [comment]
> Should I review this now?

It is never wrong to review, especially when it was requested

**Comment by tomalec** [comment]
Assuming you haven't reviewed it yet, I decided to put some more upgrades for the demo to this PR, to avoid branchitis.

**Comment by tomalec** [comment]
You definitely can review it now :)

File: index.html [hunk 1]
```diff
@@ -4,331 +4,534 @@
...
+        </p>
+    </header>
+    <div>
+        <section>
+            <h2><a href="https://github.com/Starcounter/Underwear.css">Underwear.css</a></h2>
+            <p>Usually, it's recommended to put some <a href="https://github.com/Starcounter/Underwear.css">Underwear</a> before putting the Uniform.</p>
+            <p>
+                Underwear is a minimalistic library that normalizes User Agent styles and puts a tiny layer of sugar on top. They are designed to play well together.
+                <ul>
+                    <li>Underwear unifies your light DOM content, using just native semantics,</li>
```
Comment (alshakero) [inline index.html]: I think "using just native *HTML* semantics" would be better.

File: index.html [hunk 2]
```diff
@@ -4,331 +4,534 @@
...
+                    <li>Uniform allows you to put your opinionated styling, with specific classes into shadow roots.</li>
+            </p>
+            <p>You can try how it looks with or without.</p>
```
Comment (alshakero) [inline index.html]: You can try how it looks with or without underwear

File: index.html [hunk 3]
```diff
@@ -4,331 +4,534 @@
...
-                    </label>
-                </form>
+        </section>
+        <section>
+            <h2>Buttons</h2>
+            <code>.uni-*-button-wrapper</code>
+            <p>Makes a specific type of a button</p>
+            <p>
+                We would love to use just <code>.uni-*-button</code> class,
+                but we have to use <code>-wrapper</code>  suffixand demand an additional wrapping element to support polyfilled browsers.
```
Comment (alshakero) [inline index.html]: suffixand > suffix and

File: index.html [hunk 4]
```diff
@@ -4,331 +4,534 @@
...
-                <span>3</span>
-                <span>4</span>
-                <span>5</span>
-                <span>6</span>
-                <span>7</span>
-                <span>8</span>
-                <span class="next">Next</span>
+            <div>
+                Put Underwear.css
+                <lable><input type="radio" checked name="underwear-shadow" value="on">on</label><label><input type="radio" name="underwear-shadow" value="off">off</label>
```
[the remark this answers is not in the document]
Comment (tomalec) [reply index.html]: I want to have " Put underwear on" "Put underwear off" to correspond with paragraph above. It's also the showcase of how radio buttons may work with underwear. The fact they are barely clickable is the way it is in underwear.

File: index.html [hunk 5]
```diff
@@ -4,331 +4,534 @@
...
+                            <td>
+                                <button>Default</button>
+                            </td>
+                            <td>
+                                <button disabled>Disabled</button>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td class="uni-primary-button-wrapper">
+                                <button>Primary</button>
```
[the remark this answers is not in the document]
Comment (tomalec) [reply index.html]: Agree, but this PR is about making the demo of what we have not about fixing and implementing everything what's missing in uniform. I consciously don't want to touch `uniform.css`file. Then, given a demo of current state of affairs, making ad reviewing such fixes would be waay much easier

File: index.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (alshakero) [reply index.html]: Good point. Maybe create an issue about it?

File: index.html (hunk 5 above)
[the remark this answers is not in the document]
Comment (alshakero) [reply index.html]: Yes. I apologize. We can take care of it later.

File: index.html [hunk 6]
```diff
@@ -4,331 +4,564 @@
...
+    <!-- Toggle it on and off -->
+    <link rel="stylesheet" href="../underwear.css/underwear.css">
+    <!--
+        We strongly suggest to use Uniform in shadow roots,
+        then the outer tree could be free of opinionated classes
+    -->
     <link rel="stylesheet" href="uniform.css">
-    <link rel="stylesheet" href="assets/style.css">
+
+    <!-- As a user of uniform you don't need any of belowstyles & imports -->
```
Comment (alshakero) [inline index.html]: `<!-- As a user of uniform you don't need any of the styles & imports below -->`

File: index.html (hunk 5 above)
[the remark this answers is not in the document]
Comment (tomalec) [reply index.html]: No reason to apologize, better to catch than to miss, let's track it at https://github.com/Starcounter/uniform.css/pull/20

File: index.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (tomalec) [reply index.html]: https://github.com/Starcounter/uniform.css/issues/28

**Pull request merged**